ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-SDO-nonprod.json

```diff
@@ -77,5 +77,41 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "STANDARD_DIRECTION_ORDER",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "legal-advisor"
```

Comment:
Is the below authorisation needed? I think you can just put `caseworker-civil-admin` after this line like so:

```
"legal-advisor",
"caseworker-civil-admin"
```